src/components/Error/Error.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import './Error.css';
+import { gsap, Power3 } from 'gsap';
```

Comment:
We don't need this if we aren't using gsap for animations in this file

Reply:
Removed for now. Thank you!